src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/MSBuildRestoreUtility.cs

```diff
@@ -920,25 +920,34 @@ private static RestoreLockProperties GetRestoreLockProperties(IMSBuildItem specI
                 IsPropertyTrue(specItem, "RestoreLockedMode"));
         }
 
-        public static RestoreAuditProperties GetRestoreAuditProperties(IMSBuildItem specItem)
+        public static RestoreAuditProperties GetRestoreAuditProperties(IMSBuildItem specItem, IList<string> suppressionItems)
```

Comment:
Should this type take a hash set if we're creating a hashset?

Reply:
Made the change so this is using HashSets too